### DISCUSSION TITLE
ScriptReference/RaycastCommand.ScheduleBatch

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#77] [Notes]
# ScriptReference/RaycastCommand.ScheduleBatch

https://docs.unity3d.com/ScriptReference/RaycastCommand.ScheduleBatch.html

**Comment by vertxxyz** [comment]
`ScheduleBatch` internally uses a [`TempJob`](https://docs.unity3d.com/ScriptReference/Unity.Collections.Allocator.TempJob.html) allocation, so it must be [completed](https://docs.unity3d.com/ScriptReference/Unity.Jobs.JobHandle.Complete.html) within 4 frames.
